### ISSUE TITLE
NTR: Endometritis

### ISSUE THREAD

**Issue by callahantiff** [obophenotype/human-phenotype-ontology#4715]
**Preferred term label:**
Endometritis

**Synonyms**
Inflammation of the inner lining of the uterus, infection of the uterus lining, endometrial inflammation.

**Definition (free text, please give PubMed ID)**
Inflammation of the endometrium usually caused by an infection that is located in the uterine cavity. [PMID:29960704](https://www.ncbi.nlm.nih.gov/pubmed/29960704)

**Parent term (use hpo.jax.org/app)**
HP_0030126 Abnormality of the endometrium

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
**ORCID:** [0000-0002-8169-9049](https://orcid.org/0000-0002-8169-9049)

**Comment by nicolevasilevsky** [comment]
def should be: 
Inflammation of the inner lining of the uterus

add comment:
Inflammation of the endometrium usually caused by an infection that is located in the uterine cavity.

**Comment by pnrobinson** [comment]
Inflammation of the inner lining of the uterus -- best definition.
Definition above as a comment
